pkg/runctl/run_manager_test.go

```diff
@@ -1767,6 +1816,155 @@ func Test_RunManager_Log_Elasticsearch(t *testing.T) {
 			assert.Assert(t, is.Nil(param))
 		})
 	}
+
+	/**
+	 * Test: If provided indexURL at spec.logging.elasticsearch.indexURL
+	 * does not have correct format test should fail.
+	 */
+	test = "ErrorOnWrongFormattedIndexURL"
+	for _, tc := range []struct {
+		name         string
+		indexURLJson string
+	}{
+		{"indexURLWithNoScheme", `"indexURL": "testURL"`},
+		{"indexURLWithWrongFormat2", `"indexURL": "http//testURL"`},
+	} {
+		t.Run(test+"_"+tc.name, func(t *testing.T) {
+			var err error
+
+			// setup
+			pipelineRunJSON := fmt.Sprintf(fixIndent(`
+				 {
+					 "apiVersion": "steward.sap.com/v1alpha1",
+					 "kind": "PipelineRun",
+					 "metadata": {
+						 "name": "dummy1",
+						 "namespace": "namespace1"
+					 },
+					 "spec": {
+						 "jenkinsFile": {
+							 "repoUrl": "dummyRepoUrl",
+							 "revision": "dummyRevision",
+							 "relativePath": "dummyRelativePath"
+						 },
+						 "logging": {
+							 "elasticsearch": {
+								 %s
+							 }
+						 }
+					 }
+				 }`),
+				tc.indexURLJson,
+			)
+			t.Log("input:", pipelineRunJSON)
+			examinee, runCtx, _ := setupExaminee(t, pipelineRunJSON)
+			// exercise
+			err = examinee.createTektonTaskRun(runCtx)
```

Comment:
Don't use the generic name `err` to store the examinee's result, as it may quickly happen that it get's accidently overwritten in the verify block. Better name it `resultError`.